easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
I don't think it does. ```CUDA_PATH``` is not set either in the module. Could it be that ```prepend-path``` doesn't work when passing an empty path in the ```guesses.update```? ```PATH```, ```LD_LIBRARY_PATH``` and ```CPATH``` are defined correctly